packages/http-server/src/server.ts

```diff
@@ -14,13 +15,24 @@ export const createServer = <LoaderInput>(
   const server = fastify().register(fastifyAcceptsSerializer, {
     serializers: [
       {
-        regex: /json$/,
+        regex: /application\/\S*json$/,
         serializer: JSON.stringify,
       },
     ],
     default: 'application/json',
   });
 
+  server.addContentTypeParser('*', { parseAs: 'string' }, (req, body, done) => {
+    if (typeIs(req, ['application/*+json'])) {
```

Comment:
I'm looking at the typeIs docs page and it seems that doing 
```suggestion
    if (typeIs(req, ['json'])) {
```
Should do the trick. `+json` is perhaps to strict. I think this code will not match `application/json` for instance, will it?

Reply:
Unfortunately simply putting `['json']` won't work.

I'm adding some tests to make sure it works in all the known conditions.